src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -724,19 +725,30 @@ public double getAlpha() {
     protected void setAlpha(double alpha) {
         this.alpha = alpha;
     }
-    
+
+    @Override
+    public int getTipSelectionTimeout() {
+        return tipSelectionTimeout;
+    }
+
+    @JsonProperty
+    @Parameter(names = "--tip-selection-timeout", description = TipSelConfig.Descriptions.TIP_SELECTION_TIMEOUT)
+    protected void setTipSelectionTimeout(int tipSelectionTimeout) {
+        this.tipSelectionTimeout = tipSelectionTimeout;
```

Comment:
So the variable name should be `TIP_SELECTION_TIMEOUT_SEC` too?

Reply:
I'm for the `SEC` on config as well, otherwise, you need to look at the description to fin the units,
also the java lib had a `5000` sec timeout, probably due to a unit mix-up. (https://github.com/iotaledger/iota-java/issues/154)

Reply:
changed to `TIP_SELECTION_TIMEOUT_SEC`